hooks/useStore.js

```diff
@@ -11,6 +11,24 @@ const useStore = create(set => ({
       console.error(`Error: ${error}`);
     }
   },
+
+  wishList: ['9781250278197', '123456', '9781501133572'],
+
+  addToWishList: isbn => {
+    set(state => {
+      return {
+        wishList: [...state.wishList, isbn],
+      };
+    });
+  },
+  deleteFromWishList: isbn => {
+    set(state => {
+      return {
+        wishList: state.wishList.filter(item => item !== isbn),
+      };
+    });
+    console.log(isbn);
```

Comment:
You could remove the `console.log` here.

Reply:
fix